### PR TITLE
Kaminari pagination default view

### PR DESCRIPTION
**Pull request by kalashnikovisme**
## What's changed basically?

This update is not supposed to set the default pagination behavior of applications, just setting views for kaminari.

- [x] Add default Tailwind-styled pagination for [kaminari](https://github.com/kaminari/kaminari)
- [x] Use [anyway_config](https://github.com/palkan/anyway_config) for gem configuration
- [x] Refactored `Engine`class

#### About dummy application views. 

I have a plan to use these views as Tramway default views. That's dummy application views are already Tailwind-styled.

## What's changed for tramway drivers?

To use tailwind-styled pagination

*Gemfile*
```ruby
gem 'tramway'
gem 'kaminari'
```

*config/initializers/tramway.rb*
```ruby
Tramway.configure do |config|
  config.pagination = { enabled: true } # enabled is false by default
end
```

*app/views/users/index.html.haml*
```haml
= paginate @users # it will render tailwind-styled pagination buttons by default
```

Pagination buttons look like [this](https://play.tailwindcss.com/mqgDS5l9oY)

### Song for code review

It's Pentatonix this time. I know that one track is enough for the code review, so I'll prepare a little playlist next time :slightly_smiling_face: 
https://youtu.be/ywYfMlyIkeo?si=jo7rUFGw6aZzB-Kh

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -6,6 +6,8 @@ git_source(:github) { |repo| "https://github.com/#{repo}.git" }
 # Specify your gem's dependencies in tramway.gemspec.
 gemspec
 
+gem 'anyway_config'
+
```
Comment (moshinaan) [inline Gemfile:10]: why you need empty lines in gemfile?

File: lib/tramway/engine.rb [hunk 2]
```diff
@@ -7,29 +7,65 @@ class Engine < ::Rails::Engine
...
         require 'tramway/helpers/form_helper'
 
         loaded_class.include Tramway::Helpers::FormHelper
       end
     end
+
+    # :reek:NestedIterators { enabled: false }
+    # :reek:TooManyStatements { enabled: false }
+    def configure_pagination
+      ActiveSupport.on_load(:action_controller) do |_|
```
Comment (moshinaan) [inline lib/tramway/engine.rb:54]:
```suggestion
      ActiveSupport.on_load(:action_controller) do
```

File: app/views/kaminari/_paginator.html.haml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+= paginator.render do
+  %nav.pagination.flex.items-center.justify-center.space-x-1
+    = first_page_tag unless current_page.first? 
+    = prev_page_tag unless current_page.first?
+    - each_page do |page|
```
Comment (moshinaan) [inline app/views/kaminari/_paginator.html.haml:5]: maybe its better empty line before and after `each`. just to make the code easier to read

File: Gemfile (hunk 1 above)
Comment (kalashnikovisme) [reply Gemfile]: No need, you are right! Removed :slightly_smiling_face:

File: app/views/kaminari/_paginator.html.haml (hunk 3 above)
Comment (kalashnikovisme) [reply app/views/kaminari/_paginator.html.haml:5]: There is no such empty lines in default [kaminari views](https://github.com/kaminari/kaminari/blob/a399320a063b465b42fea7e79ab2feda2198e155/kaminari-core/app/views/kaminari/_paginator.html.haml), so I think we don't need them either :slightly_smiling_face:

File: lib/tramway/engine.rb (hunk 2 above)
Comment (kalashnikovisme) [reply lib/tramway/engine.rb]: Fixed

**Pull request merged**